stp_core/config.py

```diff
@@ -40,3 +40,4 @@
 MAX_SOCKETS = 16384 if sys.platform != 'win32' else None
 ENABLE_HEARTBEATS = False
 HEARTBEAT_FREQ = 5      # seconds
+ZMQ_INTERNAL_QUEUE_SIZE = 0  # messages (0 - no limit)
```

Comment:
This would fill up all the memory of the node and crash it if one of the node is not receiving. There is a reason why high water mark exists, don't undermine that.